tests/DocumentConverter.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+using DocumentPicker.Plugin.Shared;
+using Xamarin.Essentials;
+
+namespace DocumentConverter.Tests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
```

Comment:
Not a single test?
As I said: Please test this implementation:
For: 
- if an invalid file is provided
- if a scanned pdf is provided (it is a pdf, but no vector graphics inside)
- if a valid pdf is provided

try to find additional testcases!

Reply:
I just saw - this libary throws a `PdfException` in case of invalid pdfs...
https://github.com/dmester/pdftosvg.net/blob/master/src/PdfToSvg/PdfException.cs